doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-permissions-ref.adoc

```diff
@@ -0,0 +1,372 @@
+[id='security-management-rest-api-permissions-ref_{context}']
+= Permissions
+
+The Security Management REST API supports the following endpoints for managing permissions granted to the groups, roles, and users in {CENTRAL}. The Security Management REST API base URL is `\http://SERVER:PORT/{URL_COMPONENT_CENTRAL}/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
+
+[GET] /groups/{groupName}/permissions::
++
+--
+Returns all permissions granted to a specified group.
+
+.Request parameters
+[cols="15%,45%,15%,25%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`groupName`
+|Name of the group for whom you are retrieving permissions
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+{
+	"homePage": "HomePerspective",
+	"priority": -10,
+	"project": {
+		"read": {
+			"access": false,
+			"exceptions": []
+		},
+
+	},
+	"spaces": {
+		"read": {
+			"access": true,
+			"exceptions": [
+				"dsaddsa"
```

Comment:
Can we probably put here something meaningful?

Reply:
+1, @hmanwani-rh we can replace this random text "dsaddsa" with "MySpace"

Reply:
Done